dowhy/causal_refuters/dummy_outcome_refuter.py

```diff
@@ -175,6 +179,7 @@ def __init__(self, *args, **kwargs):
         self._true_causal_effect = kwargs.pop("true_causal_effect", DummyOutcomeRefuter.DEFAULT_TRUE_CAUSAL_EFFECT)
         self._bucket_size_scale_factor = kwargs.pop("bucket_size_scale_factor", DummyOutcomeRefuter.DEFAULT_BUCKET_SCALE_FACTOR)
         self._min_data_point_threshold = kwargs.pop("min_data_point_threshold", DummyOutcomeRefuter.MIN_DATA_POINT_THRESHOLD)
+        self._test_validation_split = kwargs.pop("test_validation_split", DummyOutcomeRefuter.DEFAULT_TEST_VALIDATION_SPLIT)
```

Comment:
train_fraction is a better name, since we do not really "validate" on the held out set. we estimate causal effect on it.